build.gradle

```diff
@@ -1,7 +1,7 @@
 allprojects {
 
     group = 'com.dev-smart'
-    version = '0.1.9'
+    version = '0.1.10'
```

Comment:
set this to `0.1.10-SNAPSHOT` once the PR goes though, I'll release it under 0.1.10